src/linagora.esn.calendar/app/calendar-configuration/calendar-configuration-tab-main/calendar-configuration-tab-main.pug

```diff
@@ -52,6 +52,18 @@
       span {{ 'CalDAV URL' | translate }}
     .settings-value
       esn-clipboard-url(url="$ctrl.caldavurl")
+  .calendar-link(ng-if="$ctrl.calendarSecretLink")
+    hr
+    .settings-label
+      i.mdi.mdi-link
+      span {{ 'The secret link' | translate }}
+    .settings-value
+      esn-clipboard-url(url="$ctrl.calendarSecretLink")
+    .settings-text
+      span {{'Use this address to access the iCalendar format file (.ics). Warning: It allows anyone with the address to view your calendar and all of its events.' | translate}}
```

Comment:
```suggestion
      span {{'Anyone can export this calendar using this link.' | translate}}
```

Reply:
Let's keep the text like that. "export" is another feature.

Reply:
i used the text mentionned in description